index.html

```diff
@@ -59,22 +59,36 @@ <h2>A project for first-time contributions</h2>
 
        </div>
        <!-- ________ Contributor card END ________  -->
-       
-           
+
+
         <!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
         <!-- ^^^^ make sure there is one line of space between your card start and the card end above it ^^^^ -->
         <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
         <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
-        
-         
-         
+
+        <div class="card">
+
+            <p class="name">Paula Cardoso</p>
+            <p class="contact"><i class="fa fa-twitter"></i> <a href="YOUR twitter account link">@yourHandle</a></p>
```

Comment:
Also, if you want to add your twitter account, don't forget to put a link to it in the `href` here and write your handle where it says `@yourHandle`

Reply:
thank you. i was so focused in git i didn't realize the mistakes i was making in html. :)

Reply:
it might be good when @PaulaCardoso could add a twitter handler, or email, github profile, some online presence will always help ;)
![bildschirmfoto 2017-12-22 um 10 09 48](https://user-images.githubusercontent.com/19823791/34292222-497a8806-e700-11e7-9e1b-41e1ce275a8e.png)